holmes/core/prompt.py

```diff
@@ -0,0 +1,8 @@
+from pathlib import Path
+
+
+def append_file_to_user_prompt(user_prompt: str, file_path: Path) -> str:
+    f = file_path.open("r")
```

Comment:
Please use context manager. I suspect that the files that are opened there aren't being closed.